src/View.elm

```diff
@@ -0,0 +1,18 @@
+module View exposing (..)
+
+import Html exposing (..)
+import Html.Attributes exposing (class, src)
```

Comment:
Explicitly exposing what's needed from `Attributes` for now. Makes it more clear exactly what it's for.